static/src/javascripts/projects/membership/contributions-recurring-tab.js

```diff
@@ -79,6 +122,82 @@ const cancelContribution = (): void => {
     });
 };
 
+const handleCancellationReasonChange = () => {
+    const cancelContributionSubmit = document.querySelector(
+        CANCEL_CONTRIBUTION_SUBMIT
+    );
+    const cancellationReasonSelector = document.querySelector(
+        CANCEL_CONTRIBUTION_SELECTOR
+    );
+
+    enableCancellationSubmit();
```

Comment:
As long as it isn't changed back to the default `-- select an option --` value?

Reply:
The value `-- select an option --` can't be selected because has a `disabled` value: https://github.com/guardian/frontend/pull/18422/files#diff-e25fa09fa3fc7cfd0ce9f02b80d1d72fR12

Reply:
Shiny